ext/opentelemetry-ext-dbapi/src/opentelemetry/ext/dbapi/__init__.py

```diff
@@ -128,11 +128,65 @@ def wrap_connect_(
 def unwrap_connect(
     connect_module: typing.Callable[..., any], connect_method_name: str,
 ):
+    """Disable integration with DB API library.
+        https://www.python.org/dev/peps/pep-0249/
+
+        Args:
+            connect_module: Module name where the connect method is available.
+            connect_method_name: The connect method name.
+    """
     conn = getattr(connect_module, connect_method_name, None)
     if isinstance(conn, wrapt.ObjectProxy):
         setattr(connect_module, connect_method_name, conn.__wrapped__)
 
 
+def instrument_connection(
```

Comment:
Is there a need for `wrap_connect` anymore now that we have this? It seems this allows for better control of instrumentation, and we can recommend it as the only way to instrument DBApi connections.

Reply:
Oh it seems that `wrap_connect` is the path that should be taken for auto-instrumentation, and instrument_connection should be used for programmatic correct? Similar to what we do for `flask`?

Reply:
That's right. `wrap_connect` is used to instrument the whole library, so every new connection created is instrumented, this can be used by the autoinstrumentation and also by users that want to enable that a the library level. On the other hand, `instrument_connection` allows a more granular control by enabling the instrumentation in a single connection.